src/server/game/Entities/Unit/StatSystem.cpp

```diff
@@ -86,11 +86,31 @@ void Unit::UpdateDamagePhysical(WeaponAttackType attType)
     }
 }
 
+uint32 Unit::GetCreateVigor() const
```

Comment:
maybe a name something like `GetMaxVigor` would be more appropriate, as the talents you are using below are increasing max vigor?